src/SketchBoard/index.jsx

```diff
@@ -319,7 +293,7 @@ class SketchBoard extends React.Component {
 
               this.props.onOpenNewWindow(data)
             }}
-            print={this._print}
+            print={()=>this._sketch.print(this.props.fileName)}
```

Comment:
We're going to pass the filename from the pulse360 frontend (it's a Jira ticket I think)